.github/workflows/internal-images.yml

```diff
@@ -43,10 +43,34 @@ jobs:
           #   - needs repo:status, public_repo, write:packages, delete:packages
           password: ${{ secrets.GHCR_TOKEN }}
 
-      - name: Build and push
-        run: |  # This will only push a single architecture, which is fine as we currently only support amd64
+      - name: Setup QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Setup Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Write Dockerfile for ${{ matrix.target-tag }}
+        run: |
           cat > Dockerfile <<'EOF'
           ${{ matrix.dockerfile }}
           EOF
-          docker build -t ghcr.io/tetratelabs/func-e-internal:${{ matrix.target_tag }} .
-          docker push ghcr.io/tetratelabs/func-e-internal:${{ matrix.target_tag }}
+
+      - name : Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm64 # Build and push for amd64 and arm64 architectures.
+          tags:  ghcr.io/${{ github.repository_owner }}/func-e-internal:${{ matrix.target-tag }}
```

Comment:
fyi typo: target_tag not target-tag

my guess is that if "push: true" was here it would have failed for you.